main.js

```diff
@@ -0,0 +1,11 @@
+function onButtonClick () {
+	var startNum = parseInt(document.getElementById('startNumber').value);
+    var endNum = parseInt(document.getElementById('endNumber').value);
+	var input1 = document.getElementById('input1').value;
```

Comment:
Spacing! I know I harp on code style and formatting a lot, but it's actually really important - ragged indentations like this make your code _very_ hard to read, and while it's easy to overcome in a small script like this one, if you have a _much_ larger script the inconsistencies get magnified pretty dramatically.